doc/buildah.md

```diff
@@ -0,0 +1,68 @@
+Containers Tekton Tasks
+-----------------------
+
+# `Buildah` Tekton Task
+
+The `buildah` Task is meant to build [OCI][OCI] container images without the requirement of container runtime daemon like Docker daemon using [Buildah][containersBuildah], the Task results contain the image name and the SHA256 image digest.
+
+Please, consider the usage example below:
+
+```yaml
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: task-buildah-run-
+spec:
+  pipelineRef:
+    name: task-buildah
+  params:
+    - name: IMAGE
+      value: your-image-name
+    - name: TLS_VERIFY
+      value: true
+    - name: VERBOSE
+      value: false
+  workspaces:
+    - name: source
+      persistentVolumeClaim:
+        claimName: your-pvc-name
+```
+You'll need to replace `your-image-name`  with the actual name of the image you want to build, and `your-pvc-name`  with the name of the PersistentVolumeClaim where your source code is stored.
+In case the Container Registry requires authentication, please consider the [Tekton Pipelines documentation][tektonPipelineAuth]. In a nutshell, you need to create a Kubernetes Secret describing the following attributes:
+
+```bash
+kubectl create secret docker-registry imagestreams \
+  --docker-server="image-registry.openshift-image-registry.svc:5000" \
+  --docker-username="${REGISTRY_USERNAME}" \
+  --docker-password="${REGISTRY_TOKEN}"
+```
+
+Then make sure the Secret is linked with the Service-Account running the `TaskRun`/`PipelineRun`.
```

Comment:
On the previous PR (#40) we end up repeating this part, let's please extract the container registry authentication configuration into a single `.md` file, so we can link this and future documentation to the same place.